src/state.js

```diff
@@ -143,12 +143,27 @@ const InputChatboxMessage = new Lang.Class({
                                                '',
                                                '',
                                                GObject.ParamFlags.READWRITE,
-                                               '')
+                                               ''),
+        placeholder: GObject.ParamSpec.string('placeholder',
+                                              '',
+                                              '',
+                                              GObject.ParamFlags.READWRITE,
+                                              ''),
+        multiline: GObject.ParamSpec.boolean('multiline',
+                                             '',
+                                             '',
+                                             GObject.ParamFlags.READWRITE,
+                                             false)
     },
 
     _init: function(params, spec) {
+        // Setting defaults above in Properties doesn't seem to work, so
+        // set them here
```

Comment:
I am pretty sure setting defaults there will work if you mark the properties as construct.